docs/geometry-doc.md

```diff
@@ -5,26 +5,27 @@
 ## List
 
 ### Primitives
-Primitives are object made of raw Java types such as numbers and boolean. Enums can also be included in this layer.
+Primitives are objects made of raw Java types such as numbers and boolean. Enums can also be included in this layer.
 
 #### EPoint
 It's just a point: using an `x` and a `y` position.
 
-Functions can be applied to move it around according to another point, an angle, a distance, or any combination of these 3  
+Functions can be applied to move it around relative to another point, angle, distance, or any combination of these 3.
 #### ESize
-ESize is a holder for `width` and `height`
+ESize is a holder for `width` and `height`.
 
 #### EAngle
+//TODO: Would it be worth saying where the angle is countained from? e.g. the top/12 oclock or right hand side /3 oclock
 Angles are composed of a value and an `AngleType`: degrees (from 0 to 360), radians (from 0 to 2π) and rotations (from 0 to 1). An EAngle having a value of 2 rotations being perfectly valid.
```

Comment:
//TODO: Would it be worth saying where the angle is countained from? e.g. the top/12 oclock or right hand side /3 oclock

Reply:
The angle as no "start" it's just a wrapper around Number to handle degrees, radians and rotations.

It has no start until you apply it to something. For example when calling functions such as `Circle.toListOfPoint()`, it would be a good idea to mention it when I write the KDoc. I'll keep that in mind thanks!